alerting/src/main/kotlin/org/opensearch/alerting/AlertingPlugin.kt

```diff
@@ -183,7 +187,8 @@ internal class AlertingPlugin : PainlessExtension, ActionPlugin, ScriptPlugin, R
             RestGetWorkflowAlertsAction(),
             RestGetFindingsAction(),
             RestGetWorkflowAction(),
-            RestDeleteWorkflowAction()
+            RestDeleteWorkflowAction(),
+            RestGetRemoteIndexesAction(),
```

Comment:
I went with `indexes` because we had this issue a while back in which we were moving away from `indices`.
https://github.com/opensearch-project/index-management-dashboards-plugin/issues/637